src/screens/About/About.tsx

```diff
@@ -62,57 +83,108 @@ const About = () => {
         pageDescription="Covid Act Now is a multidisciplinary team of technologists, epidemiologists, and health experts working to help Americans understand  COVID risk in their own community."
       />
       <PageContent sidebarItems={sidebarItems}>
-        <AboutHeading1>{aboutContent.pageHeader}</AboutHeading1>
-        <AboutHeading2 id="mission">{aboutContent.introHeader}</AboutHeading2>
-        <MarkdownContent source={aboutContent.introContent} />
-        <AboutHeading2 id="contact-us">
-          {aboutContent.contactUsHeader}
-        </AboutHeading2>
-        <MarkdownContent source={aboutContent.contactUsContent} />
-        <AboutHeading2 id="partners">
-          {aboutContent.partnersHeader}
-        </AboutHeading2>
-        {aboutContent.partnersContent.map((section: any, idx: number) => {
-          return (
-            <Fragment key={`Partner-section-${idx}`}>
-              <MarkdownContent source={section.copy} />
-              <StyledGridContainer
-                container
-                spacing={1}
-                alignItems="center"
-                justify="center"
-              >
-                {section.logos.map((logo: any, i: number) => (
-                  <LogoGridItem
-                    image={logo.image}
-                    url={logo.url}
-                    altText={logo.altText}
-                    key={logo.altText}
-                  />
-                ))}
-              </StyledGridContainer>
-            </Fragment>
-          );
-        })}
-        <AboutHeading2 id="who-we-serve">
-          {aboutContent.whoWeServeHeader}
-        </AboutHeading2>
-        <PartnersSectionWrapper>
-          <MarkdownContent source={aboutContent.whoWeServeContentA} />
-          <LogoGrid logos={aboutContent.governmentLogos} />
-          <MarkdownContent source={aboutContent.whoWeServeContentB} />
-        </PartnersSectionWrapper>
+        <HashWrapper id="about">
+          <AboutHeading1>{aboutContent.aboutHeader}</AboutHeading1>
+          <MarkdownContent source={aboutContent.aboutContent} />
+          <ButtonContainer>
+            <Button to="/about#team" cta="Meet our team" />
+            <Button to="/about#contact-us" cta="Contact us" />
+          </ButtonContainer>
+        </HashWrapper>
+        <HashWrapper id="mission">
+          <AboutHeading1>{aboutContent.missionHeader}</AboutHeading1>
```

Comment:
this heading (and a few others below) are probably meant to be `h2`s (`AboutHeading2`). thinking that the only h1 should be 'about us'

Reply:
That makes sense, but it wasn't clear from looking at the renders. @joshziman can you please confirm